docker-compose.yml

```diff
@@ -30,11 +30,6 @@ services:
     environment:
       - AWS_SECRET_ACCESS_KEY
       - AWS_ACCESS_KEY_ID
-      # https://docs.databricks.com/api/latest/authentication.html#generate-a-token
-      - "AIRFLOW_CONN_DATABRICKS_DEFAULT=databricks://:@dbc-caf9527b-e073.cloud.databricks.com:443?token=${DB_TOKEN}"
```

Comment:
@fbertsch what was the exact issue you were facing when you were setting the environment variable for the GCP connection string? If you were trying to set the extra's field, they get set as url-encoded params:

```
google-cloud-platform://?extra__google_cloud_platform__key_path=%2Fkeys%2Fkey.json&extra__google_cloud_platform__scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&extra__google_cloud_platform__project=airflow
```
https://airflow.apache.org/howto/connection/gcp.html#configuring-the-connection

It's still probably a lot easier to use the command-line tool.

Reply:
Right, that's what I did for the GCP connection string. However that still didn't work either :/